platforms/software/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/BasicTypeInitIntegrationTest.groovy

```diff
@@ -76,7 +76,7 @@ class BasicTypeInitIntegrationTest extends AbstractInitIntegrationSpec {
         when:
         def gitignoreFile = targetDir.file(".gitignore")
         gitignoreFile << "*.class${System.lineSeparator()}existingIgnores"
-        run('init', '--project-name', 'someApp', '--dsl', scriptDsl.id)
+        run('init', '--project-name', 'someApp', '--dsl', scriptDsl.id, '--overwrite')
```

Comment:
The existing file here is a `.gitignore` file.  Considering we generate this file, I think it is reasonable to ask (or specify in advance) about overwriting this rather than just replacing it.